docs/examples/prometheus/kube-prometheus-stack-values-tls.yaml.example

```diff
@@ -0,0 +1,15 @@
+## Version > 1.3.0
+---
+prometheus:
+  additionalPodMonitors:
+   - name: rabbitmq
+     podMetricsEndpoints:
+     - port: prometheus-tls
+       scheme: https
+       tlsConfig:
+           insecureSkipVerify: true
```

Comment:
Prometheus scrapes the Prometheus endpoint of RabbitMQ.
Can we configure Prometheus to trust the TLS certificate of RabbitMQ such that we don't need to skip SSL validation?

Reply:
that's a good point, I am not a Prometheus expert maybe @gerhard can help.

Btw, in general, it depends how you create the certificate with self-signed cert you could have:
```
Get "https://10.244.0.29:15691/metrics": x509: cannot validate certificate for 10.244.0.29 because it doesn't contain any IP SANs
```

We had the same problem while we were preparing the [TGIR S01E08: Secure public RabbitMQ clusters
](https://www.youtube.com/watch?v=mPuMBB9_MRI&t=348s)  with the chain browser --> treafik --> rabbitmq tls endpoints and we ended up to add `insecureSkipVerify`.

It is worth to investigate.